src/components/MapView/MapView.tsx

```diff
@@ -214,6 +214,10 @@ const MapView = forwardRef<MapViewHandle, ComponentProps>(
                     pitchEnabled={pitchEnabled}
                     attributionPosition={{...styles.r2, ...styles.b2}}
                     scaleBarEnabled={false}
+                    // scaleBarEnabled={false} will not work on iOS, top: -32 is to hide the scale bar on iOS
```

Comment:
```suggestion
                    // We use scaleBarPosition with top: -32 to hide the scale bar on iOS because scaleBarEnabled={false} not work on iOS
```